src/api/services/adminServiceClient.ts

```diff
@@ -278,3 +278,13 @@ export const setSiteDomainNames = async (
   });
   return response.data;
 };
+
+export const setSiteAppNames = async (
+  siteId: number,
+  appNames: string[]
+): Promise<AdminSiteDTO> => {
+  const response = await adminServiceClient.post(`/api/site/app_names?=id=${siteId}`, {
```

Comment:
/api/site/app_names?=id=${siteId}

The first = looks unnecessary, but I can see the domain names call has the same extra =. Any ideas on why?